Orange/widgets/data/owselectrows.py

```diff
@@ -427,8 +449,14 @@ def commit(self):
                             self.error(e.args[0])
                             return
 
+                    try:
+                        floats = [float(v) for v in values]
+                    except ValueError:
+                        floats = [_DEFAULT_LOCALE_NUMERIC.toDouble(v)[0]
```

Comment:
The second value in the tuple indicates whether the parsing was successful. It might be a good idea to inspect the value and show an error in parsing fails (the way it is done when parsing floats a couple of lines above).